AB#725 Mention manifest templates and examples (#64)
edgelesssys/marblerun.sh: content/docs/workflows/define-manifest.md

The Manifest is a simple JSON file that determines the key properties of your cluster: `Packages`, `Marbles`, `Secrets`, and `RecoveryKeys`.
This article describes how to define these in your `manifest.json`.

> For a working example see the manifest of the [emojivoto demo](https://github.com/edgelesssys/emojivoto/blob/main/tools/manifest.json). See also the [sample and template manifests](https://github.com/edgelesssys/marblerun/tree/master/samples).

Reviewer: @flxflx @m1ghtym0 capitalize manifest or not?

Author: I think this issue needs to be discussed in detail at some point: At the moment we don't have a unified approach to whether or not we should capitalize manifest.
Both upper-, and lower-case are used throughout at docs at about the same rate.

Reviewer: Ok, I was under the impression that it's currently mostly uppercase.

> For a working example see the manifest used in our [emojivoto demo](https://github.com/edgelesssys/emojivoto/blob/main/tools/manifest.json), or take a look at the sample and templates provided by the [Marblerun GitHub](https://github.com/edgelesssys/marblerun/tree/master/samples).

Reviewer: ```suggestion
For a working example see the manifest of the [emojivoto demo](https://github.com/edgelesssys/emojivoto/blob/main/tools/manifest.json). See also the [sample and template manifests](https://github.com/edgelesssys/marblerun/tree/master/samples).
```
short sentences, less words
